packages/storybook/.storybook/main.js

```diff
@@ -1,16 +1,16 @@
 const path = require('path');
 const webpack = require('webpack');
 
-const env = process.env.NODE_ENV || 'development'
+const env = process.env.NODE_ENV || 'development';
```

Comment:
linting working since this fix: https://github.com/mui-org/material-ui/pull/22975